dataset_eval/evaluate.py

```diff
@@ -71,7 +71,8 @@ def evaluate_dataset(eval_job):
     # TODO(roman): Also need to catch exceptions from Gaia.
     except db.exceptions.DatabaseException as e:
         logging.info("Evaluation job %s has failed!" % eval_job["id"])
-        db.dataset_eval.set_job_status(eval_job["id"], db.dataset_eval.STATUS_FAILED)
+        db.dataset_eval.set_job_status(eval_job["id"], db.dataset_eval.STATUS_FAILED,
+                                       status_msg=str(e))
```

Comment:
I was trying to make it more clear what this string is. Updated in 0998697ab72b001d3d85de742d0c44e58c2852fe.